cuba.yml

```diff
@@ -405,3 +405,23 @@ CUBA_KEYS:
     definition: Work of adhesion per unit contact area
     shape: [1]
     type: double
+
+  MAJOR:
+    definition: Major attribute of an entity.
+    shape: [8]
+    type: string
+
+  MINOR:
+    definition: Minor version of a software tool with semantic versioning.
+    shape: [8]
+    type: string
+
+  PATCH:
+    definition: Patch version of a software tool with semantic versioning.
```

Comment:
same as before, as the patch keyword can be used in othe context. 
definition: a patch attribute of an entity